### PR TITLE
Small sysprep fixes

### PR DESCRIPTION
**Pull request by mykaul**
1. Properly inject SSH key
2. No need to enforce SElinux (it is enforced already)

### CODE REVIEW

File: lago/sysprep.py [hunk 1]
```diff
@@ -56,25 +56,11 @@ def set_iscsi_initiator_name(name):
...
-    if with_restorecon_fix:
-        # Fix for fc23 not relabeling on boot
-        # https://bugzilla.redhat.com/1049656
-        extra_options += (
-            '--firstboot-command',
-            'restorecon -R /root/.ssh',
-        )
+        '--ssh-inject',
+        'root:file:%s' % key,
+    )
```
Comment (nvgoldin) [inline lago/sysprep.py]: 👍

File: lago/vm.py [hunk 2]
```diff
@@ -120,13 +120,10 @@ def bootstrap(self):
                     self.vm._spec['disks'][0]['path'],
                     [
                         sysprep.set_hostname(self.vm.name()),
-                        sysprep.set_root_password(self.vm.root_password()),
```
Comment (nvgoldin) [inline lago/vm.py]: I think we should keep this if it doesn't have any significant time penalty:
Current use case: when using plain qcow files locally(without lago-templates)
Future use case: using other image providers.
overall, I think we should relay less on 'baked' templates and do the logic inside if we want to drop lago-images one day.

**Comment by david-caro** [comment]
I remember that we did not use the ssh-inject because there were issues with selinux, I guess that it has been resolved?

**Comment by mykaul** [comment]
@david-caro it seems to be working now... Perhaps it was an EL6 issue?

**Comment by david-caro** [comment]
@mykaul might be, perfect :+1:

**Comment by mykaul** [comment]
ci merge please

**Comment by mykaul** [comment]
ci merge please

**Comment by mykaul** [comment]
ci merge pleaes

**Comment by mykaul** [comment]
ci merge please

**Comment by mykaul** [comment]
ci merge please

**Comment by nvgoldin** [comment]
ci merge please

**Comment by nvgoldin** [comment]
ci merge please

**Comment by nvgoldin** [comment]
ci test please

**Comment by nvgoldin** [comment]
ci merge please

**Comment by mykaul** [comment]
ci test please

**Comment by nvgoldin** [comment]
@mykaul  - can you rebase?

**Pull request merged**